doc/whats_new/v0.21.rst

```diff
@@ -389,6 +389,14 @@ Support for Python 3.4 and below has been officially dropped.
   :issue:`12334` by :user:`Emmanuel Arias <eamanu@eamanu.com>`,
   `Joel Nothman`_ and `Andreas Müller`_
 
+- |Enhancement| Use of `beta` parameter in
+  :func:`metrics.cluster.supervised.homogeneity_completeness_v_measure` and
+  :func:`metrics.cluster.supervised.v_measure_score` is now enabled, allowing different
+  weight to be allocated to `homogeneity` or `completeness`. If not passed,
```

Comment:
I don't think you need to discuss the default. It is documented elsewhere, and necessarily backwards compatible